PRISM_maproom/layout.py

```diff
@@ -66,6 +67,15 @@
 )
 
 
+#point_to_layer = assign("""function(feature, latlng, context){
+#    style = {
+#        radius: 5,
+#        zIndex: 1000,
+#        color: 'black',
+#        opacity: 0.7,};
```

Comment:
I finally figured out how to make the CircleMarkers stay on top even after mouseover events. Add to this `style` object the following properties:
```
pane: 'markerPane',
interactive: false
```
`pane` makes it stay on top, and `interactive` makes mousing over the marker behave the same as mousing over the polygon below it.

BTW this object should really be called `options`, not `style`. It's not a list of CSS style properties. See [doc](https://leafletjs.com/reference.html#circlemarker).